mesonbuild/interpreterbase/interpreterbase.py

```diff
@@ -419,7 +419,7 @@ def evaluate_orstatement(self, cur: mparser.OrNode) -> T.Union[TYPE_var, Interpr
         if isinstance(r, Disabler):
             return r
         if not isinstance(r, InterpreterObject):
-            raise mesonlib.MesonBugException(f'Second argument to ot ({r}) is not an InterpreterObject but {type(r).__name__}.')
+            raise mesonlib.MesonBugException(f'Second argument to ({r}) is not an InterpreterObject but {type(r).__name__}.')
```

Comment:
```suggestion
            raise mesonlib.MesonBugException(f'Second argument to or ({r}) is not an InterpreterObject but {type(r).__name__}.')
```

Reply:
what? why???

Reply:
Because that's what the function is about.

Reply:
There's also a typo of 'Firtst' -> 'First' on line 414.

Reply:
So this currently makes it very difficult to tell that "or" is the token name rather than very bad grammar. The solution is to actually distinguish it, e.g. by quoting it.